quantecon/tests/test_distributions.py

```diff
@@ -0,0 +1,54 @@
+"""
+Filename: test_distributions.py
+Author: Quentin Batista
+
+Tests for distributions.py
+
+"""
+import numpy as np
+from numpy.testing import assert_allclose
+from math import sqrt
+from quantecon.distributions import BetaBinomial
+
+
+class TestDistributions():
+    def setUp(self):
+        self.n = 100
+        self.a = 5
+        self.b = 5
+        self.test_obj = BetaBinomial(self.n, self.a, self.b)
+    
+    def test_init(self):
+        assert_allclose(self.n, self.test_obj.n)
+        assert_allclose(self.a, self.test_obj.a)
+        assert_allclose(self.b, self.test_obj.b)
```

Comment:
I should have told you this, but for equality between scalars, you can use `eq_` from `nose.tools`:

```py
from nose.tools import eq_

eq_(self.test_obj.n, self.n)
```